package.json

```diff
@@ -28,20 +28,24 @@
     "coverage-report": "nyc report --reporter=lcov",
     "coverage": "nyc --check-coverage --branches 90 --functions 90 npm run unit",
     "standard": "standard",
+    "standard-fix": "standard --fix",
     "test": "npm run standard && npm run coverage",
     "unit": "tape test/*.js"
   },
   "devDependencies": {
-    "bitcoinjs-lib": "^3.0.0",
-    "keccak": "^1.3.0",
-    "nyc": "*",
-    "standard": "*",
-    "tape": "*"
-  },
-  "peerDependencies": {
-    "bitcoinjs-lib": "^3.0.0"
+    "bitcoinjs-lib": "^5.1.7",
+    "keccak": "^3.0.0",
+    "nyc": "^15.0.0",
+    "standard": "^14.3.1",
+    "tape": "^4.13.0"
   },
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=8.0.0"
+  },
+  "dependencies": {
```

Comment:
peerDependencies -> dependencies reason?

Reply:
Hmm, if I remember correctly I was having issues with it working correctly when I was using `peerDependencies`, so I had changed it over to `dependencies` but it might not actually be an issue still. If you would prefer we can change it back to `peerDependencies`.